crates/nu-utils/src/default_files/scaffold_config.nu

```diff
@@ -7,12 +7,12 @@
 # (or import) custom commands, or run any other startup tasks.
 # See https://www.nushell.sh/book/configuration.html
 #
-# This file is loaded after env.nu and before login.nu
```

Comment:
I think this line might be more distracting than helpful for a first-time user. Since we want most people to configure stuff in here I think we should be focused on explaining `config.nu` itself here.